lib/resources/BitcoinReceivers.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'bitcoin/receivers',
+
+  includeBasic: [
+    'create', 'list', 'retrieve'
+  ],
+
+  listBitcoinTransactions: stripeMethod({
+    method: 'GET',
+    path: '/{receiverId}/transactions',
```

Comment:
The API is exposed, but after chatting with ray & christian about it, we decided there wasn't a strong story for adding it to the bindings as a top level object. They are useful as a sublist of a receiver, though so I stuck with that.